wiki/Tournaments/RMoT_Invitational/5/en.md

```diff
@@ -0,0 +1,165 @@
+| Moderators | ![][flag_NL] [Godot](/users/6466477 "Godot"), ![][flag_DE] [Niegth](/users/1991114 "Niegth"), ![][flag_US] [You Watanabe](/users/3188911 "You Watanabe") |
+| Mappool Selectors | ![][flag_TH] [GuChangeDeWorld](/users/7125579 "GuChangeDeWorld"), ![][flag_US] [Joezapy](/users/6725659 "Joezapy"), ![][flag_CA] [Meow-Mix](/users/3021634 "Meow Mix") |
+| Graphic Designers | ![][flag_MY] [Xeious](/users/5357146 "Xeious") |
+| Streamers | ![][flag_US] [\_p0ke\_](/users/5434711 "\_p0ke\_"), ![][flag_SE] [BeatingHeart](/users/4586549 "BeatingHeart"), ![][flag_PL] [Dabis](/users/509575 "Dabis"), ![][flag_FI] [Lefafel](/users/2295850 "Lefafel"), ![][flag_NL] [Mirai Subject](/users/5639709 "Mirai Subject"), ![][flag_GB] [OzzyOzborne](/users/5377173), ![][flag_RU] [Willjoy](/users/2484373) |
+| Commentators | ![][flag_SG] [Anfang](/users/9424243 "Anfang"), ![][flag_SE] [ErroTheCube](/users/4425262 "ErroTheCube"), ![][flag_MY] [Espalza](/users/8355574 "Espalza"), ![][flag_AU] [Grassy](/users/8067959 "Grassy"), ![][flag_AT] [GumalaOC](/users/5367718 "GumalaOC"), ![][flag_PH] [kristoffer09](/users/3474344 "kristoffer09"), ![][flag_PH] [SilentVoice](/users/9570552 "SilentVoice"), ![][flag_SG] [woahsia](/users/195946 "woahsia") |
+| Referees | ![][flag_PH] [Deadpulse](/users/8575527 "Deadpulse"), ![][flag_DE] [DungeonFace](/users/1991031 "DungeonFace"), ![][flag_US] [Elcheer](/users/4420014 "Elcheer"), ![][flag_PL] [HAEN24](/users/4390077 "HAEN24"), ![][flag_SG] [Kumabread](/users/7210491 "Kumabread"), ![][flag_PL] [MagBlood](/users/6178640 "MagBlood"), ![][flag_RU] [NikerVM](/users/3618495 "NikerVM"), ![][flag_RU] [Soquire](/users/3618495 "Soquire"), ![][flag_GB] [Stedoss](/users/8331546 "Stedoss"), ![][flag_US] [YoshiLover456](/users/6843383 "YoshiLover456") |
+
+## Prizes
+
+The prizes are handed out in the form of **points**; every 48 points can be redeemed for a month of osu!supporter. The winner also recieved a profile badge.
+
+| Placing       | Prize                     |
+| ------------: | :------------------------ |
+| First Place   | 96 Points & profile badge |
+| Second Place  | 48 Points                 |
+| Third Place   | 24 Points                 |
+| Fourth Place  | 12 Points                 |
+
+### Predictions
+
+Anyone was allowed to predict the outcome of the tournament until the starting date. The predictions took place on Challonge and the top three predicters were awarded one fourth the amount awarded to the players:
+
+| Placing      | Prize     |
+| -----------: | :-------- |
+| First Place  | 24 Points |
+| Second Place | 12 Points |
+| Third Place  | 6 Points  |
+| Fourth Place | 3 Points  |
+
+![][flag_GB] [Olii](/users/7683305 "Olii") won the predictions, with ![][flag_SI] [Redavor](/users/3328606 "Redavor") in second place, and ![][flag_PL] [Malson](/users/6047395 "Malson") in third place.
+
+## Players
+
+16 players are admitted into each Invitational. Players eligible for Invitational 2 are:
```

Comment:
Only players who have reached semifinals in one of the last four Invitationals get invited. Players eligible for Invitational 5 are:

or something like that